### PR TITLE
Feat 1 authentication

### PR DESCRIPTION
**Pull request by srejitk**
This Feature Request adds the following Page Styles :

1. Sign In
2. Sign Up

Signup
Signin

### CODE REVIEW

**Review by nakulsharma15 — APPROVED** [review]
Looks good to me. No changes suggested!

File: Stylesheets/Authentication/signup.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+}
+
+.glass__input {
+    background-color: var(--component-grey-01);
+    color: black;
+    font-weight: 500;
+}
+
+.glass__input::placeholder {
+    color: #414141;
```
Comment (Sameony) [inline Stylesheets/Authentication/signup.css:15]: you can use variable here for better readability

**Review by Sameony — APPROVED** [review]
very nice code, you can merge
just one minor suggestion

File: Stylesheets/Authentication/signup.css (hunk 1 above)
Comment (srejitk) [reply Stylesheets/Authentication/signup.css:15]: Thank You. Will be adding that color to my colors stylesheet and updating it.

**Pull request merged**